--- etckeeper-1.10/update-ignore.d/01update-ignore	2013-11-14 15:33:56.904100141 +0200
+++ etckeeper-1.10/update-ignore.d/01update-ignore	2014-02-03 11:58:25.578526063 +0000
@@ -146,11 +145,80 @@
 	ignore "cups/subscriptions.conf.O"
 	ignore "fake-hwclock.data"
 	nl
+
+	comment "PLD Linux addons"
+	comment "ca-certificates: generated content"
+	#ignore "certs/ca-certificates.crt"
+	#ignore "openssl/certs/*"
+	ignore "udev/hwdb*"
+
+	comment "bash-completion"
+	ignore "bash_completion.d/*"
+	ignore "bash_completion"
+
+	#comment "openldap"
+	#ignore "ldap.secret"
+
+	comment "postfix"
+	ignore "mail/*.db"
+
+	comment "rc-scripts binary locales"
+	ignore "sysconfig/locale/*/LC_MESSAGES/rc-scripts.mo"
+
+	#comment "rc-scripts"
+	#ignore "rc.d/init.d/*"
+	#ignore "rc.d/rc?.d/[SK]*"
+	#ignore "rc.d/rc.init"
+	#ignore "rc.d/rc.local"
+	#ignore "rc.d/rc.shutdown"
+	#ignore "rc.d/rc.sysinit"
+
+	comment "zsh compiled"
+	ignore "*.zwc"
+
+	#comment "sshd keys"
+	#ignore "ssh/ssh_*_key*"
+
+	#comment "bacula passwords"
+	#ignore "bacula/*-password"
+
+	#comment "chef secret"
+	#ignore "chef/encrypted_data_bag_secret"
+
+	comment "visudo tmp"
+	ignore "sudoers.tmp"
+
+	#comment "secrets"
+	#ignore "crypttab"
+	#ignore "shadow"
+	#ignore "passwd"
+
+	#comment "certificates"
+	#ignore "httpd/ssl/"
+
+	#comment "ntp keys"
+	#ignore "ntp/keys"
+
+	comment "ssh-blacklists"
+	ignore "ssh/blacklist.DSA-1024"
+	ignore "ssh/blacklist.DSA-2048"
+	ignore "ssh/blacklist.RSA-1024"
+	ignore "ssh/blacklist.RSA-2048"
+	ignore "ssh/blacklist.RSA-4096"
+
+	comment "CVS version conflicts"
+	ignore ".#*.*.*"
+
+	comment "vigr backups"
+	ignore "group.vigr*"
+	ignore "passwd.vipw*"
 	
 	comment "editor temp files"
 	ignore "*~"
 	ignore ".*.sw?"
 	ignore ".sw?"
+	ignore ".bak"
+	ignore ".BAK"
 	ignore "#*#"
 	ignore DEADJOE
 
